pkg/controller/backup_controller.go

```diff
@@ -250,6 +253,11 @@ func (c *backupController) prepareBackupRequest(backup *velerov1api.Backup) *pkg
 	// calculate expiration
 	if request.Spec.TTL.Duration > 0 {
 		request.Status.Expiration = metav1.NewTime(c.clock.Now().Add(request.Spec.TTL.Duration))
+	} else {
+		// set default backup TTL
+		if c.defaultBackupTTL > 0 {
+			request.Status.Expiration = metav1.NewTime(c.clock.Now().Add(c.defaultBackupTTL))
```

Comment:
please add unit test(s) for this new case.

Reply:
Done